src/hooks/useCacheDetection.ts

```
@@ -0,0 +1,91 @@
+export const useCacheDetection = (): CacheDetectionResult => {
+  const [hasCacheIssues, setHasCacheIssues] = useState(false);
+  const [lastIssueDetected, setLastIssueDetected] = useState<number | null>(null);
+
+  // Check for cache issues on mount and periodically
+  useEffect(() => {
+    const checkCacheIssues = () => {
+      const cacheIssue = localStorage.getItem('media-cache-issue-detected');
+      const dismissed = localStorage.getItem('cache-issue-dismissed');
+      
+      if (cacheIssue) {
+        const issueTime = parseInt(cacheIssue);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)